src/functionaltests/java/com/ericsson/ei/subscriptions/bulk/SubscriptionBulkSteps.java

```diff
@@ -0,0 +1,117 @@
+package com.ericsson.ei.subscriptions.bulk;
+
+import com.ericsson.ei.controller.model.GetSubscriptionResponse;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.apache.commons.io.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+@Ignore
+@AutoConfigureMockMvc
+public class SubscriptionBulkSteps extends FunctionalTestBase {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SubscriptionBulkSteps.class);
+
+    private static final String TEST_RESOURCES_PATH = "src/functionaltests/resources";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private MvcResult mvcResult;
+
+    private JSONArray subscriptions;
+
+    private JSONArray retrievedSubscriptions;
+
+    @Given("^file with subscriptions \"([^\"]*)\"$")
+    public void file_with_subscriptions(String subscriptionsFileName) throws Throwable {
+        String fileContent = FileUtils.readFileToString(new File(TEST_RESOURCES_PATH + subscriptionsFileName), "UTF-8");
+        subscriptions = new JSONArray(fileContent);
+    }
+
+    @When("^make a POST request with list of subscriptions to the subscription REST API \"([^\"]*)\"$")
+    public void make_a_POST_request_with_list_of_subscriptions_to_the_subscription_REST_API(String endpoint) throws Throwable {
+        mvcResult = mockMvc.perform(MockMvcRequestBuilders.post(endpoint)
+            .accept(MediaType.APPLICATION_JSON)
+            .content(subscriptions.toString())
+            .contentType(MediaType.APPLICATION_JSON))
+            .andReturn();
+    }
+
+    @When("^make a GET request with list of subscriptions names \"([^\"]*)\" to the subscription REST API \"([^\"]*)\"$")
+    public void make_a_GET_request_with_list_of_subscriptions_names_to_the_subscription_REST_API(String subscriptionsNamesList, String endpoint) throws Throwable {
+        mvcResult = mockMvc.perform(MockMvcRequestBuilders.get(endpoint + "/" + subscriptionsNamesList)
```

Comment:
Isn't the namesList provided via the body of the Request?

Reply:
I tried generate DELETE endpoint with request body via RAML, but seems it is not possible. On the demo I demonstrated this version of bulk GET and DELETE : with subscriptionNames in URL separated by comma.

Reply:
Why? Does RestAPI request require a JSON List/Array format? Then we can have it in this format: [ {subscriptionName: subA},  {subscriptionName: subB},  {subscriptionName: subC}, ....]  ,, then it would fulfill Emichafs needs,, if I understood the discussion correctly, that a list of subscriptions can be reused from the Subscription Creations for deletions of subscriptions.

Reply:
I agree with you, but how are you going to pass this JSON list to RestAPI request without using request body?

Reply:
It has been decided to handle the subscriptionNames list in RequestBody in a new Request type with same entrypoint. A new story will be created for this.